src/index.html

```diff
@@ -1,17 +1,141 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      method="post"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset class="container">
+        <legend>Personal Information</legend>
+        <label class="box"
+          >Surname:<input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+            required
+        /></label>
+
+        <label class="box">
+          Name:<input
+            type="text"
+            id="name"
+            name="name"
+            autocomplete="off"
+            required
+        /></label>
+
+        <label class="box"
+          >How old are You?:<input
+            type="number"
+            id="age"
+            name="age"
+            min="1"
+            max="100"
+            value="22"
+            autocomplete="off"
+            required
+        /></label>
```

Comment:
```suggestion
        <label class="box">
          How old are You?:
          
          <input
            type="number"
            id="age"
            name="age"
            min="1"
            max="100"
            value="22"
            autocomplete="off"
            required
        />
      </label>
```

fix it everywhere (proofread [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md))

- About code style: all siblings (including text) should be split by an empty line
- if there are nested tags write each one in the new line